### DISCUSSION TITLE
I don't understand git achievements yet

### DISCUSSION THREAD

**Discussion by phanhuyhieu2004** [community/community#116742] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Hello everyone, I need to learn about achievements on git. I hope you can help me, thank you

[a comment by an automated account is not shown]

**Comment by ItsLasa** [comment]
In the context of Git, "achievements" typically refer to various accomplishments or milestones reached during your usage of Git version control. These achievements can be related to your coding habits, collaboration with others, or mastering certain Git commands and workflows. While Git itself doesn't have a built-in system for tracking achievements, there are external tools and plugins that can help you gamify your Git usage and track your progress.

Here are some common achievements : 
https://github.com/Thinkright20/Profile-Badges

**Comment by Duk3Dum0ntF4n** [comment]
If u speak about github achievements here you can find all list of them and how to receive them:
https://github.com/4xmen/Get-Github-Achievements

**Comment by ghostinhershell** [comment]
Hey there!  👋

Thanks for posting in the GitHub Community, @phanhuyhieu2004! We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category. The Accessibility category is a place for our community to discuss and provide feedback on the digital accessibility of GitHub products. Digital accessibility means that GitHub tools, and technologies, are designed and developed so that people with disabilities can use them.

With that being said, the users who responded above each provided resources that you can use to figure out how to earn GitHub badges. I also want to note that, because of an influx of users creating low quality, nonsensical or spam posts in the Community with the aim of earning certain Achievements, we’ve made the difficult decision to disable the ability to earn Achievements in our Community. Note that this change does not affect your ability to earn badges in other projects and communities across GitHub. This change only applies to GitHub’s Community Discussions.

You can learn more about this decision in our announcement post here [Achievements will no longer be available in the Community](https://github.com/orgs/community/discussions/106536).

I've gone ahead and moved this to the correct category for you. Good luck!

**Comment by ABHi-202864** [comment]
Response #1

**Comment by ABHi-202864** [comment]
Response #2

**Comment by TheYali1** [comment]
ddsfsd

**Comment by SahrulRD-Smatel** [comment]
Thank you very much, Response #1

**Comment by SahrulRD-Smatel** [comment]
Response #2
